cmd/keytransparency-sequencer/main.go

```diff
@@ -93,7 +93,7 @@ func main() {
 		domainStorage,
 		logAdmin, mapAdmin,
 		tlog, tmap,
-		mutations, mutations,
+		mutations, mutations, mutations,
```

Comment:
This is growing a bit right now because we're transitioning from an old storage API to a new one. `mutations` implements both, which is why we need to pass it multiple times.